src/Control/Carrier/Class.hs

```diff
@@ -1,11 +1,40 @@
-{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE FlexibleInstances, FunctionalDependencies #-}
 module Control.Carrier.Class
 ( Carrier(..)
 ) where
 
 import Control.Effect.Class
+import Control.Effect.Choose (Choose(..))
+import Control.Effect.Empty (Empty(..))
+import Control.Effect.Error (Error(..))
+import Control.Effect.NonDet (NonDet)
+import Control.Effect.Reader (Reader(..))
+import Control.Effect.Sum ((:+:)(..))
+import Control.Effect.Writer (Writer(..))
```

Comment:
The payoff: we can import all of these effect modules right where we define the `Carrier` class.